### ISSUE TITLE
Argument to pass text to the GUI instance of Crow

### ISSUE THREAD

**Issue by MurzNN** [crow-translate/crow-translate#411]
**Describe the problem or limitation you are having**
I want to automate translation of the text from a clipboard via GUI Crow window (not the selected text). But if I launch the `crow` from script - it performs the translation in text mode, example:
```
$ echo "hello world" | crow -i
hello world

[ English -> English ]

hello world
```

**Describe the solution you'd like**
Will be good to add an argument to translate the text in GUI mode (eg `-g, --gui`), like it works using shortcut `Ctrl+Alt+E`

**Additional context**
The shortcut `Ctrl+Alt+E` doesn't always get the right selected text, for example if I select the text in Google Docs (in Firefox browser) via mouse, `crow` caught previously selected text instead. So the workaround that I want is to copy text to clipboard via `Ctrl+C` and run the `xclip -selection c -o | crow -i --gui` command.

**Comment by Shatur** [comment]
Maybe use [D-Bus](https://github.com/crow-translate/crow-translate#d-bus-api) for it?

**Comment by MurzNN** [comment]
Hmm, good idea! But by which d-bus command I can send custom text to the app? I see only the way to get it from current selection.
And maybe add method to get it from clipboard?

**Comment by VolkMilit** [comment]
Yep, this need to be implemented application-side, but I think there is already protocol doing this?

**Comment by Shatur** [comment]
Agree, I think it's a good idea to add it. Should be a very simple change, anyone want to submit a PR?
